src/main/java/com/coveros/selenified/utilities/Listener.java

```diff
@@ -158,7 +158,7 @@ private void recordResult(ITestResult result) {
         String htmlFilename = "";
         String pdfFilename = "";
         if (outputFile != null) {
-            outputFile.finalizeOutputFile(result.getStatus());
+            outputFile.finalizeOutputFile(result.getStatus()-1);
```

Comment:
why are we subtracting 1 here and later on? seems like a magic number kind of deal

Reply:
It kind of is. If you look at itestresult and their statuses, it goes 1, 2, 3 for pass, fail, warning. Our enums are pass, fail, check, so using our enum ordinal, we can map the 2, if we knock out one. It's basically for simplicity. Do you think a comment would help? I'd use the itestresult ones, but they're not enums, just defined constants

Reply:
Comment would definitely be welcome.

Reply:
ok, I'll comment on it, makes total sense